src/models/OtrStore.js

```diff
@@ -24,6 +24,9 @@ function OtrStore(name, client) {
 OtrStore.prototype.initChan = function(args) {
 	var network_id = args.network;
 	var chan = args.chan;
+	var otrStore = this;
+	var client = this.client;
+
 	if (chan.type === Chan.Type.QUERY) {
 		var network = _.find(this.client.networks, {id: network_id});
```

Comment:
This isn't new from this PR, but since I looked across the project before [this comment](https://github.com/astorije/shout/pull/7/files#r39458998), we'll have to change that line, now or later, to:

```js
var network = _.find(this.client.networks, {
	id: network_id
});
```